src/packages/web/page.gleam

```diff
@@ -76,6 +87,10 @@ fn package_list_item(package: PackageSummary) -> Node(t) {
       ),
       html.h2([], [external_link_text(url, package.name)]),
       html.p_text([], package.description),
+      case links {
+        [] -> html.span([], [])
+        links -> html.div([attrs.class("package-links")], links)
```

Comment:
This looks like a nav with a ul in it to me, let's do that to have semantic HTML

Reply:
The list of package links should now be semantic HTML 👍🏻